lib/WP_Auth0_Lock_Options.php

```diff
@@ -230,8 +230,7 @@ public function get_lock_options() {
 			$extraOptions["callbackURL"] = $this->get_code_callback_url();
 		}
 
-		$options_obj = array_merge( $extraOptions, $options_obj  );
-		$options_obj = array_merge( $options_obj , $extended_settings );
+		$options_obj = array_merge( $extraOptions, $options_obj, $extended_settings );
```

Comment:
In the previous snippet you change `merge` to `replace` why does this one stay as merge?

Reply:
@cocojoe - Array dimensions, recursive vs. not recursive. `array_merge_recursive` can create arrays out of strings, whereas `array_replace_recursive` overrides. `array_merge` also overrides but it only goes 1 level deep. Looks like the options arrays are built differently.